sourcegraph/about
# Bug fixes and improvements

### Changelog
- Restores `generateChangelogItems.go` for generating changelog items for release posts. This closes #5709. Also updated the handbook to reflect this in sourcegraph/handbook#4787
- Fixes a discrepancy with our Background component where multiple utility classes would be applied based on multiple conditions being true. This caused a utility class conflict noticed on the old `CaseStudyLayout` for the `CtaSection`. Now they are scoped to their respective conditions in preparation for moving towards more code-based gradients and utility classes for backgrounds. This will help with refactoring when we remove our bg images.
- Removed another duplicate cta section (`RequestDemoForm`) in lieu of our new `CtaSection`
- Added a `scrollToAnchor` hook which closes #5637
- Updated `caniuse-lite` from a lint warning suggestion

### Test

1. Ensure prettier has standardized the proposed changes.
2. Ensure anchor links smooth scroll to the top of their element (example: `/terms/security#managed-sourcegraph-instances`)
3. Ensure CtaSection on old case study layouts have white text on a black background (example: `/case-studies/cloudflare-accelerates-debugging-and-improves-security`)
